packages/angular/cli/commands/version-impl.ts

```diff
@@ -84,8 +84,9 @@ export class VersionCommand extends Command<VersionCommandSchema> {
     if (!__dirname.match(/node_modules/)) {
       let gitBranch = '??';
       try {
-        const gitRefName = '' + child_process.execSync('git symbolic-ref HEAD', {cwd: __dirname});
-        gitBranch = path.basename(gitRefName.replace('\n', ''));
+        const gitRefName = ''
+          + child_process.execSync('git rev-parse --abbrev-ref HEAD', {cwd: __dirname});
```

Comment:
Might as well clean this up alittle while here:
`const gitRefName = child_process.execSync('git rev-parse --abbrev-ref HEAD', { cwd: __dirname, encoding: 'utf8', stdio: 'pipe' });`
Removes the need for the string concatenation conversion trick and stops any error messages from showing up on the user's console.

Reply:
👍 Done!